### PR TITLE
MKM 8 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/caught_red_handed.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name:Caught Red-Handed
+ManaCost:4 R
+Types:Instant
+A:SP$ GainControl | ValidTgts$ Creature | LoseControl$ EOT | Untap$ True | AddKWs$ Haste | SubAbility$ DBSuspect | SpellDescription$ Gain control of target creature until end of turn. Untap that creature. It gains haste until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/caught_red_handed.txt:4]: static missing

File: forge-gui/res/cardsfolder/upcoming/cerebral_confiscation.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Cerebral Confiscation
+ManaCost:2 B
+Types:Sorcery
+A:SP$ Charm | Choices$ DBDiscard,YouChoose
+SVar:DBDiscard:DB$ Discard | ValidTgts$ Opponent | NumCards$ 2 | Mode$ TgtChoose | SpellDescription$ Target opponent discards two cards
+SVar:YouChoose:DB$ Discard | ValidTgts$ Player | Mode$ RevealYouChoose | DiscardValid$ Card.nonLand | NumCards$ 1 | SpellDescription$ Target player reveals their hand. You choose a nonland card from it. That player discards that card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cerebral_confiscation.txt:6]: doesn't match oracle

File: forge-gui/res/cardsfolder/upcoming/leyline_of_the_guildpact.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:Leyline of the Guildpact
+ManaCost:GW GU BG RG
+Types:Enchantment
+K:MayEffectFromOpeningHand:FromHand
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/leyline_of_the_guildpact.txt:4]: incomplete

File: forge-gui/res/cardsfolder/upcoming/chalk_outline.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Chalk Outline
+ManaCost:3 G
+Types:Enchantment
+T:Mode$ ChangesZoneAll | ValidCards$ Card.YouOwn | Origin$ Graveyard | Destination$ Any | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever one or more creature cards leave your graveyard, create a 2/2 white and blue Detective creature token, then investigate. (Create a Clue token. It's an artifact with "{2}, Sacrifice this artifact: Draw a card.")
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/chalk_outline.txt:4]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/leyline_of_the_guildpact.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+SVar:FromHand:DB$ ChangeZone | Defined$ Self | Origin$ Hand | Destination$ Battlefield | SpellDescription$ If CARDNAME is in your opening hand, you may begin the game with it on the battlefield.
+S:Mode$ Continuous | EffectZone$ All | Affected$ Permanent.nonLand+YouCtrl | SetColor$ All | Description$ Each nonland permanent you control is all colors.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/leyline_of_the_guildpact.txt:6]: what's with that EffectZone?

**Review by tool4ever — DISMISSED** [review]

File: forge-gui/res/cardsfolder/upcoming/caught_red_handed.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+Name:Caught Red-Handed
+ManaCost:4 R
+Types:Instant
+R:Event$ Counter | ValidCard$ Card.Self | ValidSA$ Spell | Layer$ CantHappen | Description$ This spell can't be countered. (This includes by the ward ability.)
+A:SP$ GainControl | ValidTgts$ Creature | LoseControl$ EOT | Untap$ True | AddKWs$ Haste | SubAbility$ DBSuspect | SpellDescription$ Gain control of target creature until end of turn. Untap that creature. It gains haste until end of turn.
+SVar:DBSuspect:DB$ AlterAttribute | Attributes$ Suspected | Defined$ Targeted | SpellDescription$ Suspect it. (It has menace and can't block.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/caught_red_handed.txt:6]: Probably StackDescription$ SpellDescription ?

File: forge-gui/res/cardsfolder/upcoming/cease_desist.txt [hunk 7]
```diff
@@ -0,0 +1,17 @@
+Name:Cease
+ManaCost:1 BG
+Types:Instant
+A:SP$ ChangeZone | TargetMin$ 0 | TargetMax$ 2 | TargetsFromSingleZone$ True | Origin$ Graveyard | Destination$ Exile | ValidTgts$ Card | SubAbility$ DBGainLife | SpellDescription$ Exile up to two target cards from a single graveyard. 
+SVar:DBGainLife:DB$ GainLife | ValidTgts$ Player | LifeAmount$ 2 | SubAbility$ DBDraw | SpellDescription$ Target player gains 2 life and draws a card.
+SVar:DBDraw:DB$ Draw | Defined$ ParentTarget
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/cease_desist.txt:6]: TargetedPlayer might be a better Defined?

File: forge-gui/res/cardsfolder/upcoming/cease_desist.txt [hunk 8]
```diff
@@ -0,0 +1,17 @@
+Name:Cease
+ManaCost:1 BG
+Types:Instant
+A:SP$ ChangeZone | TargetMin$ 0 | TargetMax$ 2 | TargetsFromSingleZone$ True | Origin$ Graveyard | Destination$ Exile | ValidTgts$ Card | SubAbility$ DBGainLife | SpellDescription$ Exile up to two target cards from a single graveyard.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/cease_desist.txt:4]: Seems like this would need a TgtPrompt

**Review by tool4ever — APPROVED** [review]

**Pull request merged**